src/flow/darcy_flow_lmh.cc

```diff
@@ -1362,6 +1530,21 @@ DarcyLMH::~DarcyLMH() {
 }
 
 
+/// Helper method fills range (min and max) of given component
```

Comment:
Check usage of this and the next method. Seems that it was used only in DarcyFlowMH.